src/tmpfiles/tmpfiles.c

```diff
@@ -2307,17 +2408,21 @@ static int clean_item_instance(Item *i, const char* instance) {
                         sx.stx_ino != ps.st_ino;
         }
 
-        log_debug("Cleanup threshold for %s \"%s\" is %s",
+        if (age_by_format(i->age_by, &age_by) < 0)
```

Comment:
maybe name the variable differently, given that it is something different (even if it's just a different representaion of the same)